test/jasmine/tests/plot_api_test.js

```diff
@@ -348,7 +384,45 @@ describe('Test plot api', function() {
                 expect(gd.data[0].zmax).toBe(10);
 
                 expect(gd.data[0].contours).toEqual({start: 3, end: 9, size: 1});
-            });
+            })
+            .catch(fail)
+            .then(done);
```

Comment:
The biggest irony is my commit message there:

`fix the tests I must have only thought I had run successfully`